Document proxy authentication support

k-tamura opened the pull request:
Document the proxy authentication support for proxy-mappings introduced by KEYCLOAK-10659.

See also:
https://github.com/keycloak/keycloak/pull/6111

k-tamura committed:
Document proxy authentication support

Document the proxy authentication support for proxy-mappings introduced by KEYCLOAK-10659.

andymunro commented on server_installation/topics/network/outgoing.adoc:
```diff
@@ -74,6 +74,13 @@ To determine the proxy for an outgoing HTTP request the target hostname is match
 hostname patterns. The first matching pattern determines the proxy-uri to use.
 If none of the configured patterns match for the given hostname then no proxy is used.
 
+If the proxy server requires authentication, include the proxy user's credentials like `username:password@`, e.g.:
```
Instead of "like" please use "in this format"
Also, always replace "e.g."  with For example.  And please make it a new sentence.

k-tamura replied:
OK. I understand.

andymunro reviewed (changes requested):
I added single comments.

k-tamura commented:
Updated and rebased as @andymunro pointed out:

```
-If the proxy server requires authentication, include the proxy user's credentials like `username:password@`, e.g.:
+If the proxy server requires authentication, include the proxy user's credentials in this format `username:password@`.
+For example:
```
